### DISCUSSION TITLE
Supabase Starter Template not Deploying on Vercel

### DISCUSSION THREAD

**Discussion by ajayvignesh01** [vercel/community#3305] [Bug Report]
### Page to Investigate

https://vercel.com/new/templates/next.js/supabase

### Steps to Reproduce

Is anyone able to successfully deploy this template on Vercel?

I was able to do it a few days ago, but now the build fails on a fresh template. 

I went straight through Vercel website, creating a git rep, linking supabase, and clicking deploy.

Build Log:
[message.txt](https://github.com/vercel/community/files/12098966/message.txt)

My Steps:
[https://drive.google.com/file/d/13OWxtZl34vedFc4AZbb8frKz8w46wiaX/view?usp=sharing](url)

**Comment by amyegan** [comment]
Thanks for reporting this bug, @0x-Legend. I let the team know and we'll look into the cause of the error.

**Reply by ajayvignesh01** [reply]
Thank you!

**Comment by dijonmusters** [comment]
PR to fix: https://github.com/vercel/next.js/pull/52916

**Comment by ajayvignesh01** [comment]
I've added to packages.json:

```TypeScript
"dependencies": {
    "encoding": "latest"
    }
```

This got rid of one of the lines of error regarding encoding.
